File: resolucoes/php/lista01/ex005.php
<?php 

/* Faça um programa que leia as duas notas de um aluno em uma matéria e mostre na tela a sua média na disciplina. exc:

Nota 1: 4.5
Nota 2: 8.5
A média entre 4.5 e 8.5 é igual a 6.5
*/

$nota1 = (float)readline('Digite sua nota A: ');
$nota2 = (float)readline('Digite sua nota B: ');

$total = ($nota1 + $nota2)/2;

echo "\rA média entre $nota1 e $nota2 é igual a $total";

File: resolucoes/java/lista01/src/main/java/org/example/Ex005.java
/* Faça um programa que leia as duas notas de um aluno em uma matéria e mostre na tela a sua média na disciplina. ex:
```
Nota 1: 4.5
Nota 2: 8.5
A média entre 4.5 e 8.5 é igual a 6.5
```*/

package org.example;

import java.util.Scanner;

public class Ex005 {
    public static void main(String[] args) {
        Ex005.main();

    }

    public static void main() {

        Scanner sc = new Scanner(System.in);
        System.out.print("Nota 1: ");
        Float nota1 = sc.nextFloat();
        System.out.print("Nota 2: ");
        Float nota2 = sc.nextFloat();
        System.out.printf("A média entre %s e %s é igual a %s", nota1, nota2, (nota1 + nota2) / 2);
        sc.close();

    }

}
